tests/std/tests/P0896R4_views_drop/test.cpp

```diff
@@ -16,6 +16,23 @@ using namespace std;
 
 #pragma warning(disable : 6011) // Dereferencing NULL pointer '%s'
 
+struct evil_convertible_to_difference {
+    evil_convertible_to_difference() = default;
+    evil_convertible_to_difference(const evil_convertible_to_difference&) {
+        throw(42);
+    }
+    evil_convertible_to_difference(evil_convertible_to_difference&&) = default;
+    evil_convertible_to_difference& operator                         =(const evil_convertible_to_difference&) {

```

Comment:
Style nitpick: A newline would avoid clang-format's ugly results here. Also occurs in `P0896R4_views_take/test.cpp`.